bench.py

```diff
@@ -37,48 +47,46 @@ def calculator(test, arg, n_runs, only_result, workpath=os.getcwd()):
             statistics.median(w),
             statistics.pstdev(w)),
         bold=True)
-    os.remove(os.path.join(workpath, "Benchmark.txt"))
 
+    if uploadresult is True:
```

Comment:
``if uploadresult`` looks better. If you absolutely want to compare it to ``True``, you can use ``uploadresult == True``, it's better than using ``is`` because python does not guarantee that ``True`` is a singleton, as far as I know.